config/locales/hyrax.zh.yml

```diff
@@ -8,6 +8,7 @@ zh:
       messages:
         conflict: 您的更改无法保存，因为另一用户(或后台作业）在您开始编辑后更新了%{model}。 请确保所有文件附件已成功完成之后再试一次。这已经是最新刷新后保存的副本%{model}
   blacklight:
+    application_name: must be here for features page to open
```

Comment:
Haven't tracked that down yet, but it was definitely the case. I pushed it up this way and am looking into that next.